File: PRAK602-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>

int main(){
    int ruangan;
    printf(" ");
    scanf("%d", &ruangan);

    int zetsu[ruangan];
    printf(" ");
    for (int i = 0; i < ruangan; i++){
        scanf("%d", &zetsu[i]);
    }
    for (int i = 0; i < ruangan; i++){
        printf("%d ", zetsu[i] * (i + 1));
    }
    return 0;
}

File: PRAK602-2310817310008-MUHAMMADRIZKIRAMADHAN.py
ruangan = int(input(" "))
zetsu = []
nilai = input((" ").split())
for i in range(0, ruangan):
    zetsu.append(int(nilai[i]))
for i in range(0, ruangan):
    print(zetsu[i] * (i + 1), end=" ")